account_move_update_analytic/README.rst

```diff
@@ -0,0 +1,80 @@
+============================
+Account move update analytic
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--analytic-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-analytic/tree/14.0/account_move_update_analytic
+    :alt: OCA/account-analytic
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-analytic-14-0/account-analytic-14-0-account_move_update_analytic
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/87/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows users to update analytic account and tags on account_move lines
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+1. In a posted move, go to the move line you want to update, and click on 'Update analytic' to open the wizard. (You also have the button directly on account move lines)
+2. Current analytic is shown, you can select another one then click on Confirm
+3. Analytic lines will be dropped from current analytic account and recreated on new analytic account/tags.
+
```

Comment:
Please add this images in this PR o the next one. Thank you! (You can use the github URL)

![imagen](https://user-images.githubusercontent.com/8736623/207829639-5477e8e7-c768-4b26-a71e-cd16a07c84df.png)

![imagen](https://user-images.githubusercontent.com/8736623/207829833-1031caa6-b1be-4e52-9f1c-6c82074334a7.png)